struqture-py/tests/bosons/bosonic_noise_system.rs

```diff
@@ -691,7 +691,10 @@ fn test_format_repr() {
 
         rust_system
             .add_operator_product(
-                (new_pp.as_gil_ref().into(), new_pp.as_gil_ref().into()),
+                (
+                    new_pp.as_ref().clone().into(),
+                    new_pp.as_ref().clone().into(),
```

Comment:
Given the `as_ref`, is the `clone` needed as well as the `into`?

Reply:
I was able to remove the first clone but since the `new_pp` variable is being moved, we have to at least clone the first one.
I think before there was a `clone()` hidden in the `into()` call